src/topology/continuous_on.lean

```diff
@@ -117,6 +117,9 @@ mem_inf_of_left h
 theorem self_mem_nhds_within {a : α} {s : set α} : s ∈ 𝓝[s] a :=
 mem_inf_of_right (mem_principal_self s)
 
+theorem eventually_mem_nhds_within {a : α} {s : set α} : ∀ᶠ x in 𝓝[s] a, x ∈ s :=
+self_mem_nhds_within
```

Comment:
@urkud Thanks for your help. Is there a reason you added this lemma?

Reply:
To be able to use `eventually.mono` instead of `mem_of_superset`.